modules/fscloud/variables.tf

```diff
@@ -46,6 +46,16 @@ variable "schemas" {
   default     = []
 }
 
+variable "schema_global_rule" {
+  type        = string
+  description = "Schema global compatibility rule. Only allowed for enterprise plan, must be '' for other plans. Allowed values are 'NONE', 'FULL', 'FULL_TRANSITIVE', 'FORWARD', 'FORWARD_TRANSITIVE', 'BACKWARD', 'BACKWARD_TRANSITIVE'."
```

Comment:
The fscloud module only supports enterprise, so you can remove the text `Only allowed for enterprise plan, must be '' for other plans`

Reply:
I have removed the line as a whole as we are doing plan validation for this already. Also removed the TF variable validaion.